YTPMVE.py

```diff
@@ -123,6 +120,7 @@ def exitScript(errMessage, exitCode, retryWithPythonScript):
     YTPMVE_file.write(str(note_starts[i][0])+",")#Save first argument, note start time
     YTPMVE_file.write(str(note_durations[i])+"\n")#Save second argument, note duration
 
+YTPMVE_file.close()
+ErrorLogFile.close()
```

Comment:
these lines are not necessary as they are also ran in `exitScript()`

Reply:
Deleted these lines.